syncmaster/backend/api/v1/runs.py

```diff
@@ -117,7 +117,7 @@ async def start_run(
             type=RunType.MANUAL,
         )
 
-        log_url = Template(get_worker_settings().worker.LOG_URL_TEMPLATE).render(
+        log_url = Template(WorkerAppSettings().worker.LOG_URL_TEMPLATE).render(
```

Comment:
If this is required on backend side, and not worker, it actually should be a part of ServerAppSettings